wiki/en/Running-a-Server.md

```diff
@@ -125,6 +125,21 @@ Any settings made using the graphical interface will be stored in the `Jamulusse
 
 ---
 
+### Minimum setup to run in Registered mode
+
+You can run headless Jamulus Server on Linux using `systemctl`, which is covered in other sections.
+You can also run Jamulus Server from the command line. Enter the following command to [run a Registered Server](Running-a-Server#server-types):
+
+~~~
+jamulus-headless --nogui --server \
+        --directoryaddress hostname:port \
+        --serverinfo "yourServerName;yourCity;[country ID]"
+~~~
+
+See the table below for `hostname:port` values.
+
+**Note**: Semicolon and newline characters are not allowed in `yourServerName` and `yourCity` within the `--serverinfo` argument. 
```

Comment:
```suggestion
**Note**: Semicolon and newline characters are not allowed in `[name]` and `[ciry]` within the `--serverinfo` argument. See the [Command line options](#command-line-options) for more information on the country. 
```

The link to command line options will need to be checked, but the linkchecker will later on complain.

Reply:
Why would this local anchor work? I don't see a command-line-options keyword, and I don't see a title where these dashes are spaces.

Reply:
The linkchecker will complain and then we'd find the correct section. It should be hidden behind an import in the site and only shows up in compiled form.